local_extensions/__init__.py

```diff
@@ -11,19 +12,57 @@
 
 
 VOLTO_MIN_VERSION = 16
-VOLTO_GENERATOR_MIN_VERSION = 6
+
+
+VOLTO_GENERATOR_VERSIONS = {
+    16: (6, 8),
+    17: (8, 9),
+    18: (9, None),
+}
+
 
 DEFAULT_NODE = "18"
 
 VOLTO_NODE = {
     16: 16,
```

Comment:
Why is this one an int but the others are str?

Reply:
Fixed